functions-and-operators/cast-functions-and-operators.md

```diff
@@ -6,16 +6,67 @@ aliases: ['/docs/dev/functions-and-operators/cast-functions-and-operators/','/do
 
 # Cast Functions and Operators
 
-Cast functions and operators enable conversion of values from one data type to another. TiDB supports all of the [cast functions and operators](https://dev.mysql.com/doc/refman/5.7/en/cast-functions.html) available in MySQL 5.7.
+Cast functions and operators enable conversion of values from one data type to another. TiDB supports all of the [cast functions and operators](https://dev.mysql.com/doc/refman/8.0/en/cast-functions.html) available in MySQL 8.0.
 
 ## List of cast functions and operators
 
 | Name                                     | Description                      |
 | ---------------------------------------- | -------------------------------- |
-| [`BINARY`](https://dev.mysql.com/doc/refman/8.0/en/cast-functions.html#operator_binary) | Cast a string to a binary string |
+| [`BINARY`](https://dev.mysql.com/doc/refman/8.0/en/cast-functions.html#operator_binary) | Cast a string to a binary string. |
```

Comment:
I think either all should end with a period, or none?

Reply:
```suggestion
| [`BINARY`](https://dev.mysql.com/doc/refman/8.0/en/cast-functions.html#operator_binary) | Cast a string to a binary string |
```